### DISCUSSION TITLE
Low root space on Orange Pi 5 Plus

### DISCUSSION THREAD

**Discussion by Mohsen7s** [immortalwrt/immortalwrt#1397] [Q&A]
### Describe the bug

The root partition is limited to 300M while there is 250GB of unallocated free space on emmc. 

### ImmortalWrt version

Snapshot

### ImmortalWrt release

SNAPSHOT

### ImmortalWrt target/subtarget

rockchip/armv8

### Device

Orange Pi 5 Plus

### Image kind

Official downloaded image

### Steps to reproduce

Get report using df -Th

### Actual behaviour

Root partition didnt claim free space on emmc

### Expected behaviour

Root partition should claim whole emmc free space.

### Additional info

Tried to resize with multiple packages without luck.

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by cvb011** [comment]
You can manually creat a new partition formatted as ext4, then copy the existing file in /overlay to it and mount it as /overlay, this will solve the issue. (Note that the setting of immortalwrt WILL BE RESET) .

**Comment by KardRi** [comment]
so where is your df -Th ? no log = no solution
